src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,37 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Pageblock Refresh
+Pageblocks have a refresh option to automatically renew the pageblock data without having to reload the entire page. 
+This is useful for pageblocks that display data that changes frequently, such as a list of recent records, an update to a chart, or a list of recent comments.
```

Comment:
I'd consider changing `list of current records` to a more real example, such as `list of new leads`